cmd/axon/sql.go

```diff
@@ -88,8 +113,9 @@ func insertRow(conn *sqlx.DB, schema string, change *warppipe.Changeset) error {
 	query, args := prepareInsertQuery(schema, change)
 	_, err := conn.NamedExec(query, args)
 	if err != nil {
-		return fmt.Errorf("Error: failed to insert: %v", err)
+		return fmt.Errorf("failed to insert %s: %w", change, err)
 	}
+	log.Printf("row inserted: %s", change)
```

Comment:
These may be too noisy. Perhaps should be a roll up reported every 10-30 seconds? See #1